chapter_07/7-8.py

```diff
@@ -0,0 +1,12 @@
+sandwich_orders = ["tuna", "meatball", "reuben", "italian", "dogfish"]
+finished_sandwiches = []
+
+while sandwich_orders:
+    current_sandwich = sandwich_orders.pop()
+
+    print(f"I made your {current_sandwich.title()} sandwich.")
+    finished_sandwiches.append(current_sandwich)
+
+print("\nFinished sandwiches:\n")
+for sandwich in finished_sandwiches:
+    print(sandwich.title())
```

Comment:
Doesn't affect how the script works, but for formatting it's customary to have an empty line at the end of the file, makes it easier to read/scroll, especially with longer files. Make sure the empty line doesn't have any whitespace chars.

Reply:
Looks like that depends on the version of PEP you're using. This bot likes them removed.

Reply:
I will continue to add an empty line at the end of files in that case.